README.md

```diff
@@ -1,2 +1,18 @@
-# bgp-agent
-Cloud-native BGP
+# Ligato BGP Agent
+
+`Ligato BGP Agent` is a BGP information provider. It provides BGP information in an unified format to allow to retrieve BGP information from different sources(different BGP frameworks, eg: GoBGP, ExaBGP) and supporting multiple extensions (different AFI/SAFIS ex: IPV4 / Unicast, IPV6 / Unicast).
+## Architecture
+
+The architecture of the `Ligato BGP Agent` is shown in the following figure.
+
+![arch](docs/imgs/bgpagent.png "High Level Architecture of BGP Agent")
```

Comment:
Please, we need to see there:
1. gobpg logo
2. multiple watchers
3. aligned exabgp/gobgp